src/App.tsx

```diff
@@ -1,5 +1,5 @@
 import "./App.css";
-import { TodoList } from "./TodoList";
+import { TodoList } from "./TodoListMain";
```

Comment:
Let's change the name of this file and the component in the file to `Dashboard`.